includes/class-wc-payments.php

```diff
@@ -1633,6 +1636,19 @@ public static function init_woopay() {
 		}
 	}
 
+	/**
+	 * Load stripe site messaging script.
+	 *
+	 * @return void
+	 */
+	public static function load_stripe_site_messaging() {
+		if ( WC_Payments_Features::is_bnpl_affirm_afterpay_enabled() ) {
```

Comment:
Is this the right way to check for feature flag? I guess I'll have to update #6518 and even simplify it 😀

Reply:
Yes this introduced at some point.